compass/conf/CoordinatorConfiguration.java

```diff
@@ -56,6 +56,12 @@
   @Parameter(names = "-validator", description = "Validator nodes to use")
   public List<String> validators = new ArrayList<>();
 
+  @Parameter(names = "-validationAttempts", description = "If tips validation fails, obtain new tips and validate up to this number of attempts.")
+  public int validationAttempts = 10;
+
+  @Parameter(names = "-validationDelay", description = "Obtained tips might not be solid right away on the validators, sleep the specified amount of seconds before validating.")
```

Comment:
is it seconds? milliseconds?
A better description please

Reply:
It's already written "sleep the specified amount of seconds before validating.". More specific than this?

Reply:
couldn't see it on the github screen :+1: